models/clan_test.go

```diff
@@ -444,7 +444,7 @@ var _ = Describe("Clan Model", func() {
 				)
 
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(Equal(fmt.Sprintf("Clan was not found with id: %s", clan.PublicID)))
+				Expect(err.Error()).To(Equal(fmt.Sprintf("Forbidden action. GameId: %s, PlayerId: %s, ActionableId: %s", clan.GameID, player.PublicID, clan.PublicID)))
```

Comment:
what do you mean by `ActionableId`?

Reply:
Something that a player can take action on (Clan), and as such can be forbidden of said action. I was trying to future proof, but now I think replacing for ClanId is enough.

Reply:
Ah, ok! It's a good idea but I'd rather be explicit here. I don't see any other resources being added to Khan anytime soon.